documentation/staging/content/release-notes.md

```diff
@@ -40,6 +41,18 @@ draft: false
 
 ### Change log
 
+#### Operator 3.2.4
+
+* Added support for the sessionAffinity field for the clusterService ([#2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/2383)).
+* Moved several logging messages to the CONFIG level ([#2387](https://github.com/oracle/weblogic-kubernetes-operator/pull/2387)).
+* Resolved an issue related to scalingAction.sh when there were multiple domains in the same namespace ([#2388](https://github.com/oracle/weblogic-kubernetes-operator/pull/2388)).
+* Updated introspection and other scripts to consistently use ISO-8601 timestamp formatting ([#2386](https://github.com/oracle/weblogic-kubernetes-operator/pull/2386)).
```

Comment:
I thought the operator log's timestamp format also changed in 2386?

Reply:
Updated